path = /QUIC-FormalVerification/QUIC-Ivy/doc/examples/quic/test/temp/346
implementation directory: /QUIC-FormalVerification/quic-implementations/quic-go/client/
implementation command: ./client -X /QUIC-FormalVerification/tls-keys/secret.log -P -v 127.0.0.1 4443 
quic_client_test_0rtt_invalid
[<utils.IvyTest.IvyTest object at 0x7effa9f820b8>]
./client -X /QUIC-FormalVerification/tls-keys/secret.log -P -v 127.0.0.1 4443 
/QUIC-FormalVerification/QUIC-Ivy/doc/examples/quic/build/quic_client_test_0rtt_invalid (46) ...
/QUIC-FormalVerification/QUIC-Ivy/doc/examples/quic/test/temp/346/quic_client_test_0rtt_invalid46.out
/QUIC-FormalVerification/QUIC-Ivy/doc/examples/quic/test/temp/346/quic_client_test_0rtt_invalid46.err
/QUIC-FormalVerification/QUIC-Ivy/doc/examples/quic/test/temp/346/quic_client_test_0rtt_invalid46.iev
implementation command: /bin/sleep 5; exec ./client -R -X /QUIC-FormalVerification/tls-keys/secret.log -P -v 127.0.0.1 4443 
quic_process pid: 13178
iclient = 0
quic_client_test_0rtt_invalid
timeout 30 /QUIC-FormalVerification/QUIC-Ivy/doc/examples/quic/build/quic_client_test_0rtt_max seed=831 the_cid=1 server_port=4443 ;sleep 1;timeout 30 /QUIC-FormalVerification/QUIC-Ivy/doc/examples/quic/build/quic_client_test_0rtt_invalid seed=831 the_cid=1 server_port=4444 
command is timeout 30 /QUIC-FormalVerification/QUIC-Ivy/doc/examples/quic/build/quic_client_test_0rtt_max seed=831 the_cid=1 server_port=4443  from quic_client_test_0rtt_invalid

client return code: 1
iclient = 0
quic_client_test_0rtt_invalid
timeout 30 /QUIC-FormalVerification/QUIC-Ivy/doc/examples/quic/build/quic_client_test_0rtt_max seed=322 the_cid=1 server_port=4443 ;sleep 1;timeout 30 /QUIC-FormalVerification/QUIC-Ivy/doc/examples/quic/build/quic_client_test_0rtt_invalid seed=322 the_cid=1 server_port=4444 
command is timeout 30 /QUIC-FormalVerification/QUIC-Ivy/doc/examples/quic/build/quic_client_test_0rtt_invalid seed=322 the_cid=1 server_port=4444  from quic_client_test_0rtt_invalid

client return code: 1
pid is in use
quic_process.kill()
FAIL
False
     Run  ... version_not_found
0      0  ...               0.0
1      1  ...               0.0
2      2  ...               0.0
3      3  ...               0.0
4      4  ...               0.0
..   ...  ...               ...
341  341  ...               0.0
342  342  ...               0.0
343  343  ...               0.0
344  344  ...               0.0
345  345  ...               0.0

[346 rows x 28 columns]
error: 1 tests(s) failed
